src/frontend/src/utils/userNumber.test.ts

```diff
@@ -0,0 +1,81 @@
+import { getUserNumber, setUserNumber } from "./userNumber";
+
+testLocalStorage("user number can be set", () => {
+  expect(getUserNumber()).toBe(undefined);
+  setUserNumber(BigInt(123456));
```

Comment:
`BigInt(123456)` is repeated few don't know if you rather like to have a common const value for your test?

Reply:
Yeah was thinking about this, but I'll wait until I have to repeat it a few more times :)